scripts/us_census/pep/population_estimate_by_race/preprocess.py

```diff
@@ -174,117 +201,134 @@ def _clean_county_80_xls_file(df: pd.DataFrame, file_path: str) -> pd.DataFrame:
     # "10":"45-49 year olds","11":"50-54 year olds","12":"55-59 year olds",
     # "13":"60-64 year olds","14":"65-69 year olds","15":"0-74 year olds",
     # "16":"75-79 year olds","17":"80-84 year olds","18":"85 years old and older"}
-    df['Total People']=df[1]+df[2]+df[3]+df[4]+df[5]+df[6]+df[7]+\
-        df[8]+df[9]+df[10]+df[11]+df[12]+df[13]+df[14]+df[15]+\
-        df[16]+df[17]+df[18]
-    df['FIPS'] = [f'{x:05}' for x in df['FIPS']]
-    df['Info'] = df['Year'].astype(str) + '-' + df['FIPS'].astype(str)
-    df.drop(columns=['Year','FIPS',1,2,3,4,5,6,7,8,9,10,11,12,13,14\
-        ,15,16,17,18],inplace=True)
-    df = df.groupby(['Info','Race/Sex']).sum().transpose().\
-        stack(0).reset_index()
-    df['Year'] = df['Info'].str.split('-', expand=True)[0]
-    df['geo_ID'] = "geoId/" + df['Info'].str.split('-', expand=True)[1]
-    # Deriving the total values as per the requires SV's
-    df['Total']=df['White male']+df['White female']+df['Black male']\
-        +df['Black female']+df['Other races male']+df['Other races female']
-    df['White Alone'] = df['White male'] + df['White female']
-    df['Black or African American Alone'] = df['Black male'] + \
-        df['Black female']
-    df.drop(columns=['Info','level_0','White male','White female','Black male'\
-        ,'Black female','Other races male','Other races female'],inplace=True)
-    final_df = pd.DataFrame()
-    final_df = pd.concat([final_df, df])
-    df['geo_ID'] = df['geo_ID'].str[:-3]
-    df = df.groupby(['Year','geo_ID']).sum().\
-        reset_index()
-    df.drop(columns=['Total'], inplace=True)
-    df_temp = pd.DataFrame()
-    df_temp = pd.concat([df, df_temp])
-    df_temp['geo_ID'] = "country/USA"
-    df_temp = df_temp.groupby(['Year','geo_ID']).sum().\
-        reset_index()
-    final_df = pd.concat([final_df, df, df_temp])
-    final_df = final_df.drop_duplicates()
+    try:
+        df['Total People']=df[1]+df[2]+df[3]+df[4]+df[5]+df[6]+df[7]+\
+            df[8]+df[9]+df[10]+df[11]+df[12]+df[13]+df[14]+df[15]+\
+            df[16]+df[17]+df[18]
+        df['FIPS'] = [f'{x:05}' for x in df['FIPS']]
+        df['Info'] = df['Year'].astype(str) + '-' + df['FIPS'].astype(str)
+        df.drop(columns=['Year','FIPS',1,2,3,4,5,6,7,8,9,10,11,12,13,14\
+            ,15,16,17,18],inplace=True)
+        df = df.groupby(['Info','Race/Sex']).sum().transpose().\
+            stack(0).reset_index()
+        df['Year'] = df['Info'].str.split('-', expand=True)[0]
+        df['geo_ID'] = "geoId/" + df['Info'].str.split('-', expand=True)[1]
+        # Deriving the total values as per the requires SV's
+        df['Total']=df['White male']+df['White female']+df['Black male']\
```

Comment:
Do we use 'Total' in the output? If so, can we add 'dcAggregate/' prefix to the measurementMethod?

Reply:
No we don't use "Total" in output 
This is for SV level total in the code 
prefix to the measurementMethod is not required here